README.md

```diff
@@ -1 +1,53 @@
-# github-mirror
+# GitHub Mirror
+
+Implements [conditional requests](https://developer.github.com/v3/#conditional-requests)
+
```

Comment:
I think it would help a bit more detail 
* explaining that this is a server that sits between github and the client that caches the results. 
* having a note explaining this only affects `GET` request while it lets pass the rest of the methods
* telling that cache is done in memory and per process for the moment

Reply:
Indeed. Will work on that.